home/i18n/zh-cn/docusaurus-plugin-content-docs/current/help/nvidia.md

```diff
@@ -0,0 +1,37 @@
+---
+id: nvidia  
+title: 监控：NVIDIA监控      
```

Comment:
Adding a few spaces makes it easier to read and organize the document.

Reply:
> Adding a few spaces makes it easier to read and organize the document.

@yuluo-yx  already midified